modules/API/external/external_api.js

```diff
@@ -622,9 +629,29 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
             case 'local-storage-changed':
                 jitsiLocalStorage.setItem('jitsiLocalStorage', data.localStorageContent);
 
+                // Since this is internal event we don't need to emit it to the consumer of the API.
+                return true;
+            case 'config-overwrite': {
```

Comment:
Won't we need the same logic when a user changes the setting? When we add it, that is. In that case, doesn't it make more sense to emit an event when the PiP setting changes? Overriding the config should effectively change the setting.

Reply:
Which setting do you mean? 

Do you mean the setting in the UI for disabling PiP? We had before but then decided that the consumer can actually implement this on their own with overriding the config flag and that we won't implement it in jitsi-meet for now.